### PR TITLE
Helpers: Add argument null check to StartPopupWatcher

### PR DESCRIPTION
**Pull request by dlujic**
StartPopupWatcher originally throws generic null exception, added validation of parameters that are not null. Null check from other function refactored to reduce code duplication.

Also added a test to cover the case where null arguments are passed, and refactored some common actions from test methods into Setup and TearDown.

### CODE REVIEW

File: AutomationHelpers/src/UserCodeCollections/PopupWatcherLibrary.cs [hunk 1]
```diff
@@ -39,6 +39,9 @@ public static class PopupWatcherLibrary
         [UserCodeMethod]
         public static PopupWatcher StartPopupWatcher(RepoItemInfo findElement, RepoItemInfo clickElement)
         {
+            CheckArgumentNotNull(findElement, "findElement");
```
Comment (ahoisl) [inline AutomationHelpers/src/UserCodeCollections/PopupWatcherLibrary.cs]: How can we require >C#6.0 in the long run? C#5.0 is included in .NET 4.5, but for C#6.0 and above you need the MSBuild Tools to be separately installed.

Comment (ahoisl) [reply AutomationHelpers/src/UserCodeCollections/PopupWatcherLibrary.cs]: Even if Studio supports C#6.0, you might not have a C#6 compiler installed. So we would need to also cope with that... But not a discussion for this thread

File: AutomationHelpers/test/PopupWatcherLibraryTests.cs [hunk 2]
```diff
@@ -70,7 +70,13 @@ public void StartPopupWatcherTest_Twice_ThrowsException()
         public void StartPopupWatcherTest_WithoutParameters_ThrowsException()
         {
             //Assert
```
Comment (ahoisl) [inline AutomationHelpers/test/PopupWatcherLibraryTests.cs]: Remove this line

**Comment by ahoisl** [comment]
Changed base path to integration -> once we have a few changes, we merge to master and also set the new package version

File: AutomationHelpers/test/PopupWatcherLibraryTests.cs (hunk 2 above)
Comment (dlujic) [reply AutomationHelpers/test/PopupWatcherLibraryTests.cs]: Removed!
